lib/src/internal/FlowManager.cpp

```diff
@@ -186,22 +208,51 @@ namespace mxl::lib
             // Write the json file to disk.
             writeFlowDescriptor(tempDirectory, flowDef);
 
-            auto flowData = std::make_unique<ContinuousFlowData>(makeFlowDataFilePath(tempDirectory).string().c_str(), AccessMode::CREATE_READ_WRITE);
+            auto flowData = std::unique_ptr<ContinuousFlowData>{};
+            try
+            {
+                flowData = std::make_unique<ContinuousFlowData>(makeFlowDataFilePath(tempDirectory).string().c_str(), AccessMode::CREATE_READ_WRITE);
+            }
+            catch (std::exception const& e)
+            {
+                MXL_ERROR("FlowManager: Failed to create continuous flow data for '{}': {}", uuidString, e.what());
+                throw std::runtime_error(std::string("FlowManager: Failed to mmap continuous flow data: ") + e.what());
+            }
 
             auto& info = *flowData->flowInfo();
             info.version = 1;
             info.size = sizeof info;
 
             info.common = initCommonFlowInfo(flowId, flowFormat);
 
-            info.continuous = {};
+            info.continuous = ContinuousFlowInfo{};
```

Comment:
Please keep the original implicit form, or is there a specific reason why that needed to be replaced?